### PR TITLE
Add qplot for trajectory

### PR DESCRIPTION
**Pull request by MonikaFu**
Related to #264 

In this PR I add a `quickplot()` prototype for trajectory.

### CODE REVIEW

File: R/qplot_trajectory.R [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      plot.subtitle = element_text(
+        hjust = 0.5, vjust = 0.5,
+        size = base_size * 10 / 12
+      )
+    ) +
+    labs(
+      title = glue("Production trajectory of {to_pretty_label(data$technology[1])} technology
+                   in the {to_title(data$sector[1])} sector"),
+      subtitle = glue("The coloured areas indicate trajectories in reference to a scenario.
+                      The red area indicates trajectories not aligned with any sustainble scenario."),
```
Comment (maurolepore) [inline R/qplot_trajectory.R:50]: Best to wrap at 80 characters. To continue a sentence beyond that `glue()` supports `\\`.

File: R/qplot_trajectory.R [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    # TODO: move to theme_2dii()
+    theme(
+      plot.subtitle = element_text(
+        hjust = 0.5, vjust = 0.5,
+        size = base_size * 10 / 12
+      )
+    ) +
+    labs(
+      title = glue("Production trajectory of {to_pretty_label(data$technology[1])} technology
+                   in the {to_title(data$sector[1])} sector"),
```
Comment (maurolepore) [inline R/qplot_trajectory.R:48]: The title does not end in full stop but the subtitle does. Is this intentional?

**Review by maurolepore — APPROVED** [review]
I added my suggestions as a PR into this one.

File: R/qplot_trajectory.R (hunk 2 above)
Comment (MonikaFu) [reply R/qplot_trajectory.R]: yes

**Pull request merged**